### PR TITLE
Login/SignUp

### PR DESCRIPTION
**Pull request by hamdibm**
i added  the login and sign up pages

### CODE REVIEW

**Comment by Hamdi-BM** [comment]
i added sign up and login pages

File: src/app/auth/login/page.tsx [hunk 1]
```diff
@@ -1,7 +1,71 @@
...
+  disbaled: "#D9D9D9"
 }
+const Login = () => {
+  
+  const router = useRouter();
+
+  const handleSignUp = () => {
+    router.push('/auth/signup');
+  };
+  // const [isClient, setIsClient] = useState(false)
```
Comment (onlymachiavelli) [inline src/app/auth/login/page.tsx:18]: Remove this Comment

File: src/app/auth/signup/page.tsx [hunk 2]
```diff
@@ -1,7 +1,55 @@
...
+import { Input, Link, Button } from '@nextui-org/react';
 
-export default Page
+const SignUp = () => {
+  const router = useRouter();
+
+  const handleLogin = () => {
+    router.push('/auth/login');
+  };
+  // const [isClient, setIsClient] = useState(false)
```
Comment (onlymachiavelli) [inline src/app/auth/signup/page.tsx:13]: Remove this comment

File: src/app/page.tsx [hunk 3]
```diff
@@ -1,12 +1,50 @@
...
+import SignUp from './auth/signup/page';
+
 const App = () => {
+  const [isClient, setIsClient] = useState(false)
+  useEffect(() => {
+    setIsClient(true)
+  }, [])
+  const router = useRouter();
+  const pathname=usePathname();
+  const handleTabChange = (newKey: string | number) => {
```
Comment (onlymachiavelli) [inline src/app/page.tsx:15]: don't use pathname

File: src/app/page.tsx [hunk 4]
```diff
@@ -1,12 +1,50 @@
-"use client"
-import {} from "react"
-import { Button } from "@nextui-org/react"
```
Comment (onlymachiavelli) [inline src/app/page.tsx:3]: this page is reserved for the landing page nor login or signup or other thing

**Pull request closed without merging**